_translator-files/po/de/Client-Troubleshooting.po

```diff
@@ -45,228 +47,230 @@ msgid ""
 
 #. type: Plain text
 #: ../wiki/en/Client-Troubleshooting.md:35
 msgid "### You all sound OK, but it's difficult to keep together"
-msgstr ""
+msgstr "### Der Sound ist gut aber es ist schwierig im Takt zu bleiben"
 
 #. type: Plain text
 #: ../wiki/en/Client-Troubleshooting.md:37
 #, no-wrap
 msgid "**If at all possible, DO NOT listen to your direct signal.** Make sure you are listening as much as you can to the sound of your own instrument/voice _coming back from the Server_. This signal has you and your playing partners mixed together in sync, and will be the signal that your bandmates are hearing too. So listening to that means you will be in sync with each other (assuming you all have reasonably low latency). Note that if one or more musicians are not following this rule, they will slow down as they play or sing.\n"
-msgstr ""
+msgstr "**Wenn möglich, höre NICHT auf dein direktes Signal.** Stelle sicher, dass du so viel wie möglich auf den Klang deines eigenen Instruments/Stimme hörst, der _vom Server_ zurückkommt. Dieses Signal wird von dir und deinen Mitspielern synchron gemischt und ist das Signal, das auch deine Bandkollegen hören. Wenn du dieses Signal hörst, seid ihr untereinander synchron (vorausgesetzt, alle haben eine einigermaßen geringe Latenz). Beachte, dass ein oder mehrere Musiker, die sich nicht an diese Regel halten, beim Spielen oder Singen langsamer werden.\n"
 
 #. type: Plain text
 #: ../wiki/en/Client-Troubleshooting.md:39
 msgid "You can test whether you are hearing your signal correctly by doing the following:"
-msgstr ""
+msgstr "So kannst du testen, ob du dein Signal richtig hörst:"
 
 #. type: Plain text
 #: ../wiki/en/Client-Troubleshooting.md:41
 msgid "Run Jamulus and connect to a Server with a long ping time (greater than 200ms)"
-msgstr ""
+msgstr "Starte Jamulus und stelle eine Verbindung zu einem Server mit einer langen Ping-Zeit (mehr als 200 ms) her"
 
 #. type: Plain text
 #: ../wiki/en/Client-Troubleshooting.md:42
 msgid "Clap your hands once (or play one short note on an instrument). You should hear the sound from the Jamulus Server come back, but significantly delayed."
-msgstr ""
+msgstr "Klatsche einmal in die Hände (oder spiele einen kurzen Ton auf einem Instrument). Du solltest hören, dass der Ton vom Jamulus Server zurückkommt, allerdings deutlich verzögert."
 
 #. type: Plain text
 #: ../wiki/en/Client-Troubleshooting.md:43
 msgid "Click the \"Mute\" button under your Jamulus name/slider in the main Jamulus mixer window (**NOT** the \"Mute Myself\" button on the left)."
-msgstr ""
+msgstr "Klicke im Hauptfenster des Jamulus-Mixers auf die Schaltfläche \"MUTE\" unter deinem Jamulus-Namen/Slider (**NICHT** die Schaltfläche \"Stummschalten\" auf der linken Seite)."
```

Comment:
Backslashes... @ignotus666 anything to change here?

Reply:
The '\"' is needed to show " on the page text
Ok for me